README.md

```diff
@@ -76,6 +127,9 @@ spec:
     path: .
     plugin:
       name: argocd-vault-plugin
+      env:
+        - name: AVP_PATH_PREFIX # Optional: Can be added to resource or a env var via a kubernetes secret
+          value: some/path/to/vault
     repoURL: <repo>
```

Comment:
Small nit but let's avoid using the AVP placeholder syntax in places we don't mean to show an example of using the AVP placeholders. So maybe `repoURL: http://your-repo/`